src/core/ledger/distributionSummary/allocationSummary.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import {Ledger} from "../ledger";
+import {type Distribution} from "../distribution";
+import {type Allocation} from "../grainAllocation";
+import {type GrainReceipt} from "../grainAllocation";
+import * as G from "../grain";
+import sortBy from "../../../util/sortBy";
+import {type IdentityId} from "../../identity";
+import * as NullUtil from "../../../util/null";
+import {type CurrencyDetails} from "../../../api/currencyConfig";
+import {formatCenter} from "./distributionSummary";
+
+export function allocationMarkdownSummary(
+  distribution: Distribution,
+  allocation: Allocation,
+  ledger: Ledger,
+  currencyDetails: CurrencyDetails
+) {
+  const {suffix: currencySuffix} = currencyDetails;
+  const {receipts} = allocation;
+  const totalDistributed: G.Grain = getTotalDistributed(receipts);
+  console.log(`## ${allocation.policy.policyType} Policy`);
+
+  console.log(
+    `### Distributed ${G.format(totalDistributed, 0, currencySuffix)} to ${
+      receipts.length
+    } identities`
+  );
```

Comment:
Maybe just add a total line to the bottom of the table?
(I dont think the # of identities would fit, but that's ok, i dont think its super important.)